lib/dialects/abstract/query-generator.js

```diff
@@ -145,7 +145,7 @@ module.exports = (function() {
             - allowNull: Boolean
     */
     /* istanbul ignore next */
-    changeColumnQuery: function(tableName, attributes) {
```

Comment:
I tend to agree. I'd say the best thing here is to ignore the specific error in this file. Is that fine for you?